src/native/libs/System.Globalization.Native/pal_icushim.c

```diff
@@ -446,7 +448,7 @@ static void InitializeVariableMaxAndTopPointers(char* symbolVersion)
     ucol_setVariableTop_ptr = (ucol_setVariableTop_func)GetProcAddress((HMODULE)libicui18n, symbolName);
 #else
     char symbolName[SYMBOL_NAME_SIZE];
-    sprintf(symbolName, "ucol_setVariableTop%s", symbolVersion);
+    snprintf(symbolName, SYMBOL_NAME_SIZE, "ucol_setVariableTop%s", symbolVersion);
```

Comment:
>SYMBOL_NAME_SIZE [](http://example.com/codeflow?start=25&length=16)

this should be `MaxICUVersionStringWithSuffixLength + 1` too.

Reply:
Same as the above case.